src/gosling-brush/linear-brush-model.ts

```diff
@@ -42,7 +42,7 @@ export class LinearBrushModel {
     private data: LinearBrushData;
 
     /* drag */
-    private startEvent: typeof D3Selection.event;
+    private startEvent: any; // D3Drag.D3DragEvent.sourceEvent which is `any`
```

Comment:
could we just make this explicit?

```suggestion
    private startEvent: D3Drag.D3DragEvent.sourceEvent;
```

Although it is `any`, if it were to ever change we would automatically get the correct type.